src/pages/in/embed/_id/button.vue

```diff
@@ -335,6 +342,19 @@ export default {
     onClickFrontDisplayName() {
       logTrackerEvent(this, 'LikeButtonFlow', 'clickFrontDisplayName', 'clickFrontDisplayName', 1);
     },
+    handleWindowMessage(event) {
+      if (event.orgin !== `https://${LIKE_CO_HOSTNAME}`) return;
+      if (event.data) {
+        const { data } = event;
+        switch (data.action) {
+          case 'LOGGED_IN':
+            window.location.reload();
```

Comment:
what is the use of `reload` here?

Reply:
on second thought... seems calling `updateUser()` is enough

Reply:
but if u are trying to solve cookie problem, u probably still wont get cookie in 3rd party iframe?